styleguide.config.js

```diff
@@ -131,5 +135,18 @@ module.exports = {
   },
   title: 'Mavenlink Design System',
   usageMode: 'expand',
+  updateExample(props = {}, exampleFilePath) {
```

Comment:
What is going on here? :)

Reply:
I wanted to make it easier for developers to understand how to apply custom class names to our components.

https://mavenlink.github.io/design-system/icon-v2/#/Components/IconButton

In these examples, I have "an example stylesheet" which is used in the code examples. Hopefully, it is easier to connect the dots on how we are customizing the components (with `composes` and all!)